xml/security_vpnserver.xml

```diff
@@ -264,12 +264,53 @@ cd /etc/openvpn
       <para>
        Bridging is a more complex solution. It is recommended when you need
        to browse Windows file shares across the VPN without setting up a
-       Samba or WINS server. Bridged VPN is also needed to use
-       non-IP protocols (such as IPX) or applications relying on network
+       Samba or WINS server. Bridged VPN uses network TAP devices, that
+       simulate a link layer between the VPN endpoints and operate in layer
+       2 carrying Ethernet frames. This makes it possible to use non-IP
+       protocols (such as IPX) or applications relying on network
```

Comment:
This makes it possible to use non-IP protocols such as IPX or applications (i know this is existing content , if you can please make this change)